File: src/main/java/varun/db/connection/SprinbootJDBC/FoodInfo.java
package varun.db.connection.SprinbootJDBC;

public class FoodInfo {
	private int id;
	private String cuisine;
	private String foodName;
	private int calories;
	private int counter;
	private int rating;
	
	public FoodInfo(String cuisine, String foodName, int calories, int counter, int rating) {
		this.cuisine = cuisine;
		this.foodName = foodName;
		this.calories = calories;
		this.counter = counter;
		this.rating = rating;
	}
	
	public FoodInfo() {}
	
	public int getID() {
		return this.id;
	}
	
	public void setID(int id) {
		this.id = id;
	}
	
	public String getCuisine() {
		return this.cuisine;
	}
	
	public void setCuisine(String cuisine) {
		this.cuisine = cuisine;
	}
	
	public String getFoodName() {
		return this.foodName;
	}
	
	public void setFoodName(String foodName) {
		this.foodName = foodName;
	}
	
	public int getCalories() {
		return this.calories;
	}
	
	public void setCalories(int calories) {
		this.calories = calories;
	}
	
	public int getCounter() {
		return this.counter;
	}
	
	public void setCounter(int counter) {
		this.counter = counter;
	}
	
	public int getRating() {
		return this.rating;
	}
	
	public void setRating(int rating) {
		this.rating = rating;
	}
	
	@Override
	public String toString() {
		String desc = "Food Info [ " + id + " ," + cuisine + " ," + foodName +" ," + calories;
		return desc;
	}
}

File: dev/src/Components/FoodInfo.js
import React,  { useState, useEffect, Component, Text } from "react";

import Avatar from "@material-ui/core/Avatar";
import Button from "@material-ui/core/Button";
import CssBaseline from "@material-ui/core/CssBaseline";
import TextField from "@material-ui/core/TextField";
import { Link } from "react-router-dom";
import Grid from "@material-ui/core/Grid";
import GroupIcon from "@material-ui/icons/Group";
import Typography from "@material-ui/core/Typography";
import { makeStyles } from "@material-ui/core/styles";
import Container from "@material-ui/core/Container";
import axios from "axios";


export default function FoodInfo(){
  const [firstLoad, setLoad] = React.useState(true);

  const [cuisine, setCuisine] = React.useState("");
  const [foodName, setFoodName] = React.useState("");
  const [calories, setCalories] = React.useState(0);
  const [counter, setCounter] = React.useState(0);
  const [rating, setRating] = React.useState(0);


  const handleCuisineChange = event => setCuisine(event.target.value);
  const handleFoodNameChange = event => setFoodName(event.target.value);
  const handleCaloriesChange = event => setCalories(event.target.value);
  const handleCounterChange = event => setCounter(event.target.value);
  const handleRatingChange = event => setRating(event.target.value);


  // Get JSON info from the HTTP 
  async function DisplayFunc(toInput){
    const response = await fetch
  }


}

